frontend/packages/pipelines-plugin/src/components/repository/consts.ts

```diff
@@ -0,0 +1,19 @@
+export enum RepositoryFields {
+  REPOSITORY = 'Repository',
+  BRANCH = 'Branch',
+  URL_REPO = 'RepoUrl',
+  URL_ORG = 'RepoOrg',
+  SHA = 'sha',
```

Comment:
This is fine being called SHA as it's the technical id of the label. It however doesn't need to be echo'ed to the user as such. (this is a comment continuation of a previous comment -- nothing to do here)